Sources/AblyChat/AblyCocoaExtensions/InternalAblyCocoaTypes.swift

```diff
@@ -164,7 +164,7 @@ internal final class InternalRealtimeClientAdapter<Underlying: ProxyRealtimeClie
                 }
             }.get()
         } catch {
-            throw error.toInternalError()
+            throw InternalError.ablyCocoa(error)
```

Comment:
`fromAblyCocoa`? it's not immidiately obvious what is being converted to what here